styles/images.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Anurag Sharma">
+    <title>images</title>
+    <script src="https://kit.fontawesome.com/f45a42e7ea.js" crossorigin="anonymous"></script>
```

Comment:
why is there a need to add font awesome script tag? could have done via css

Reply:
moved it to utility.css